tests/unit/query_helpers/test_query_helper.py

```diff
@@ -0,0 +1,41 @@
+"""Unit tests for query helper class."""
+
+from ols.src.query_helpers import QueryHelper, camel_to_snake
+from ols.utils import config
+
+
+def test_camel_to_snake():
+    """Test camel case to snake case conversion."""
+    assert camel_to_snake("") == ""
+    assert camel_to_snake("dog") == "dog"
+    assert camel_to_snake("LOUD") == "loud"
+    assert camel_to_snake("quiet") == "quiet"
+    assert camel_to_snake("OCP4") == "ocp4"
+    assert camel_to_snake("Is there a space?") == "is there a space?"
+    assert camel_to_snake("rainyDay") == "rainy_day"
+    assert camel_to_snake("NiceDay") == "nice_day"
+    assert camel_to_snake("AGoodDay") == "a_good_day"
```

Comment:
Yes, this can be parametrized, but I found this better readable. I can change it though.